docs/source/operator/credentials.rst

```diff
@@ -0,0 +1,56 @@
+.. _frankConsoleCredentials:
+
+Credentials
+-----------
+
+Frank configurations can communicate with external systems. These external systems may require credentials like usernames and passwords. The Frank!Framework needs to know these credentials. It is your job to configure these credentials, because they should not be included in Frank configurations. This section explains how you can do this.
+
+First, you have to know the *alias* of the external account. This is a name used in the Frank configuration to reference the credentials. The Frank config uses strings like ``${credential:username:alias1}`` and ``${credential:password:alias1}``. Here, ``alias1`` is the alias of the external account being accessed. Then you have to provide the credentials of the alias. The Frank!Framework allows you to do this in many ways. We demonstrate credentials first by providing them in a properties file.
```

Comment:
Er kan ook op veel plekken een `authAlias` attribuut gebruikt worden

Reply:
Dit heb ik toegevoegd. Ik hou het materiaal over Frank configuraties kort omdat deze tekst voor operators is.